packages/core/src/routes-me/verification-code.ts

```diff
@@ -33,12 +42,30 @@ export default function verificationCodeRoutes<T extends AuthedMeRouter>(
   router.post(
     '/verification-codes/verify',
     koaGuard({
-      body: object({ email: string().regex(emailRegEx), verificationCode: string().min(1) }),
+      body: object({
+        email: string().regex(emailRegEx),
+        verificationCode: string().min(1),
+        action: union([literal('changeEmail'), literal('changePassword')]),
```

Comment:
Will it be better to define an enum type for action?

Reply:
How can an enum be better than string union? 😂

Reply:
enum can be uses as values, say you want to iterate all possibilities: `Object.values()` but not possible for string union

Reply:
Yeah, but in this case they are the same.